App/UI/Home/Cells/HomeServiceActiveCell.swift

```diff
@@ -184,12 +193,18 @@ class HomeServiceActiveCell: UICollectionViewCell, ModellableView, ReusableView,
       .minHeight(53)
       .bottom(HomeView.cellHorizontalInset)
 
+    self.discoverMoreButton.pin
+      .left(HomeView.cellHorizontalInset)
+      .sizeToFit(.widthFlexible)
```

Comment:
I'd rather give a left and right margin and use the whole width so we have better control of the hit area of the button.